Modules/_winapi.c

```diff
@@ -464,6 +474,41 @@ _winapi_CreateFile_impl(PyObject *module, LPCTSTR file_name,
     return handle;
 }
 
+/*[clinic input]
+_winapi.CreateFileMappingW -> HANDLE
+
+    file_handle: HANDLE
+    security_attributes: LPSECURITY_ATTRIBUTES
+    protect: DWORD
+    max_size_high: DWORD
+    max_size_low: DWORD
+    name: LPCWSTR
+    /
+[clinic start generated code]*/
+
+static HANDLE
+_winapi_CreateFileMappingW_impl(PyObject *module, HANDLE file_handle,
+                                LPSECURITY_ATTRIBUTES security_attributes,
+                                DWORD protect, DWORD max_size_high,
+                                DWORD max_size_low, LPCWSTR name)
+/*[clinic end generated code: output=c6b017501c929de1 input=35cadabe53b3b4da]*/
+{
+    HANDLE handle;
+
+    Py_BEGIN_ALLOW_THREADS
+    handle = CreateFileMappingW(file_handle, security_attributes,
+                                protect, max_size_high, max_size_low,
+                                name);
+    Py_END_ALLOW_THREADS
+
+    if (handle == NULL) {
+        PyErr_SetFromWindowsErr(0);
```

Comment:
Consider using `PyErr_SetFromWindowsErrWithUnicodeFilename(0, name)`.

Reply:
Done -- changed in commit 68785333a25d902a42252ace3a8268357efd6094.